python-backend/app/config.py

```diff
@@ -34,7 +34,10 @@ class Config(object):
     SQLALCHEMY_MAX_OVERFLOW = 20
     SQLALCHEMY_POOL_TIMEOUT = 300
 
-
+    #Flask-uploads configs
+    UPLOADED_DOCUMENT_DEST = os.environ.get('UPLOADED_DOCUMENT_DEST', '/app/document_uploads')
+    #100MB file limit
+    FILE_BYTE_LIMIT = 100 * 1024 * 1024
```

Comment:
Same question as before. Is there a reason to convert this to bytes here, and then convert it back to MB in the code?

Reply:
It has to be in byte form for the configuration.